src/main/java/org/broadinstitute/hellbender/utils/pca/PCA.java

```diff
@@ -0,0 +1,120 @@
+package org.broadinstitute.hellbender.utils.pca;
+
+import org.apache.commons.math3.linear.Array2DRowRealMatrix;
+import org.apache.commons.math3.linear.ArrayRealVector;
+import org.apache.commons.math3.linear.RealMatrix;
+import org.apache.commons.math3.linear.RealVector;
+import org.broadinstitute.hellbender.utils.MathUtils;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.svd.SVD;
+
+import java.util.function.Function;
+import java.util.stream.DoubleStream;
+
+/**
+ * Principal component analysis.
+ *
+ * @author Valentin Ruano-Rubio &lt;valentin@broadinstitute.org&gt;
+ */
+public final class PCA {
+
+    /**
+     * Array of variable centers subtracted from the data-matrix before PCA.
+     */
+    private final double[] centers;
+
+    /**
+     * Principal component loadings (i.e. eigen-vectors).
```

Comment:
"eigen-vectors -> eigenvectors"

also, i'm not sure about the terminology, but are "loadings" typically orthonormal?  i think they are usually scaled by the square roots of the eigenvalues.  maybe just call this field "eigenvectors" and note their normalization in the doc.